app/services/error-handler.js

```diff
@@ -6,27 +6,22 @@ export default class ErrorHandlerService extends Service {
 
 @service l10n;
 
-handleError(error) {
+handle(error) {
   const errorStatus =  (error?.errors?.[0]?.status) ? parseInt(error?.errors?.[0]?.status) : error;
+  const errorMsg = error?.errors?.[0]?.detail ? error?.errors?.[0]?.detail : error;
```

Comment:
Why are you defaulting to error if ` error?.errors?.[0]?.detail` is not present?

Reply:
what should be the msg, if error?.errors?.[0]?.detail is no present.